pipelines/broad/dna_seq/germline/joint_genotyping/JointGenotyping.wdl

```diff
@@ -1,18 +1,27 @@
 version 1.0
 
-import "../../../../../tasks/broad/JointGenotypingTasks.wdl" as Tasks
```

Comment:
If you were working off dsde-pipelines the directory structure is different in WARP so you may want to use the previous one.